docs/development-setup.md

```diff
@@ -0,0 +1,23 @@
+
+
+An AEA <a href="../package-imports">consists of many packages </a>. When developing, it helps to be able to save packages in a local package registry. This guide helps setting up a local package registry and configuring the working directory for development.
+
+## Development setup
+
+To prepare a directory (henceforth working directory) for development with the AEA framework you can take a few steps:
+
+- Ensure you start with an empty working directory to avoid any unnecessary side-effects.
+
+- In your working directory, create an empty folder called `packages`. This folder will act as the local registry for packages.
+
+- In your working directory, create a `.env` file with the constant `PYTHONPATH=path_to_packages_dir` where `path_to_packages_dir` is the path to the packages folder in your working directory.
```

Comment:
two comments:
1. `PYTHONPATH=path_to_packages_dir` will overwrite the old `PYTHONPATH`, that causes problems if the project environment relied on the content of `PYTHONPATH`
2. the path should point to the _parent directory_ of the `packages` directory

Reply:
Re 2 see further down. Re 1. we need to modify it, so what do you suggest then instead.

Reply:
Btw we tested this on @jrriehl 's machine and it worked fine

Reply:
we agreed to change the instruction with `PYTHONPATH=path_to_working_dir`.

Reply:
regarding 1: try whether `PYTHONPATH=$PYTHONPATH:path_to_working_dir` works. If yes, that should be preferable.

Reply:
Yes, this works. I just checked.

Reply:
Sorry, to be more specific, what worked was `PYTHONPATH=$PYTHONPATH:path_to_packages_dir`.

The above suggestion did not work.